test/intl402/Temporal/PlainDateTime/prototype/toZonedDateTime/disambiguate-basic.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.tozoneddatetime
+description: Basic cases for disambiguate options
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(2020, 3, 8, 2, 0);
+
+assert.sameValue(
+  `${dt.toZonedDateTime("America/Los_Angeles", { disambiguation: "compatible" })}`,
```

Comment:
In this case you'll need to have a custom time zone with a DST transition, so consider using `TemporalHelpers.springForwardFallBackTimeZone`.

Reply:
Good idea -- done.